### PR TITLE
[Bug][P1] Fix key parser error

### PR DESCRIPTION
**Pull request by deerhunter4**
Try-except block was added to read_keys() function in key_reader.py.

### CODE REVIEW

File: key_reader.py [hunk 1]
```diff
@@ -7,9 +7,14 @@
...
-        key, value = line.strip().split(',')
-        keys_dict[key] = value
+    try:
+        for line in keys_file:
+            key, value = line.strip().split(',')
+            keys_dict[key] = value
+    except ValueError:
+        raise ValueError(f"""File {KEYS_PATH} has an incorrect format.
+                         Should be comma-delimited txt. E.g. the key name
+                         should be separated from the key by a comma.""")
```
Comment (KrzysiekJodlowski) [inline key_reader.py]: @deerhunter4 kudos for using the try-except block! However, this text is somewhat chaotic, you're repeating the idea of comma separation twice here. I'd suggest to bring a direct example instead, e.g.:
```
ValueError: File ./better_weather_keys.txt has an incorrect format.
It should be a comma-delimited text, following the scheme: \"<key-name>,<key-value>\"
```
Also, the console print is strangely shifted after the first line:
```
ValueError: File ./better_weather_keys.txt has an incorrect format.
                         Should be comma-delimited txt. E.g. the key name
                         should be separated from the key by a comma.
```
Can we even it out?

**Review by KrzysiekJodlowski — CHANGES_REQUESTED** [review]

File: key_reader.py (hunk 1 above)
Comment (deerhunter4) [reply key_reader.py]: @KrzysiekJodlowski I corrected the error message. However, I think that there is no easy way to print it in a more even way. With all the multi-line error messages it is the same.
```
ValueError: Entered forecast_period value is not allowed.
                         Valid options are: 24h, 48h, 72h, 7days, 10days, 14days
                         (default: 24h).
```

Comment (KrzysiekJodlowski) [reply key_reader.py]: @deerhunter4 the way it looks makes it harder to read. Please dig a bit deeper and try to find a proper solution, or, use non-multiline strings instead. Also, keep in mind we have the same situation with most of the prints we make, so it'd be good to handle them all the same way.

Comment (deerhunter4) [reply key_reader.py]: @KrzysiekJodlowski all multi-line error messages corrected.

File: key_reader.py [hunk 2]
```diff
@@ -7,16 +7,22 @@
...
-        keys_dict[key] = value
+    try:
+        for line in keys_file:
+            key, value = line.strip().split(',')
+            keys_dict[key] = value
+    except ValueError:
+        raise ValueError(f"File {KEYS_PATH} has an incorrect format.\n"
+                         "It should be a comma-delimited text, "
+                         "following the scheme:\n"
+                         "\"<key-name>,<key-value>\"")
```
Comment (KrzysiekJodlowski) [inline key_reader.py]:
```
ValueError: File ./better_weather_keys.txt has an incorrect format.
It should be a comma-delimited text, following the scheme:
"<key-name>,<key-value>"
```
I'd say let's remove the `\n` after the `scheme` word to have it a bit more concise:
```
ValueError: File ./better_weather_keys.txt has an incorrect format.
It should be a comma-delimited text, following the scheme: "<key-name>,<key-value>"
```

File: key_reader.py [hunk 3]
```diff
@@ -7,16 +7,22 @@
...
 
     keys_file.close()
 
     if 'weatherapi.com' not in keys_dict or 'meteostat.net' not in keys_dict:
-        raise KeyError("""The API key names in the better_weather_keys.txt
-                       file are missing or their names might be incorrect.
-                       The correct names are: 'weatherapi.com', 'meteostat.net'""")
+        raise KeyError("The API key names in the better_weather_keys.txt "
+                       "file are missing or their names might be incorrect. "
+                       "The correct names are: 'weatherapi.com', 'meteostat.net'")
```
Comment (KrzysiekJodlowski) [inline key_reader.py:25]:
```
KeyError: "The API key names in the better_weather_keys.txt file are missing or their names might be incorrect. The correct names are: 'weatherapi.com', 'meteostat.net'"
```
I suggest adding `\n` after the `incorrect` word to have a similar pattern as for the keys file error (if the previous suggestion is applied) and forecast period error:
```
KeyError: "The API key names in the better_weather_keys.txt file are missing or their names might be incorrect.
The correct names are: 'weatherapi.com', 'meteostat.net'"
```

File: key_reader.py (hunk 2 above)
Comment (deerhunter4) [reply key_reader.py]: @KrzysiekJodlowski Done.

File: key_reader.py (hunk 3 above)
Comment (deerhunter4) [reply key_reader.py:25]: @KrzysiekJodlowski 
This is the one example that I was mentioning during our meeting, for which '\n' is not working.
```
KeyError: "The API key names in the better_weather_keys.txt file are missing or their names might be incorrect.\n The correct names are: 'weatherapi.com', 'meteostat.net'"
```
So I am leaving it as it is now.

**Review by KrzysiekJodlowski — APPROVED** [review]

**Pull request merged**